src/pages/code-of-conduct.js

```diff
@@ -0,0 +1,238 @@
+                otherwise.
+              </li>
+            </ol>
+
+            <p className="mb-5">
+              Organizers will be happy to help members contact local law
+              enforcement, provide escorts, or otherwise assist any attendee to
+              feel safe for the duration of any event. We value your presence.
+            </p>
+
+            <p className="mb-5">When making a report, please include:</p>
+
+            <ol className="list-disc pl-6 mb-4">
+              <li>Your contact info for follow-up contact.</li>
+              <li>
+                Names (legal, nicknames, or pseudonyms) of any individuals
+                involved.
+              </li>
+              <li>
+                If there were other witnesses besides you, please try to include
+                them as well.
+              </li>
+              <li>
+                When and where the incident occurred. Please be as specific as
+                possible.
+              </li>
+              <li>Your account of what occurred.</li>
+              <li>
+                If there is a publicly available record (e.g. an email archive
+                or a public Slack log) please include a link.
+              </li>
+              <li>Any extra context you believe existed for the incident.</li>
+              <li>If you believe this incident is ongoing.</li>
+              <li>Any other information you believe we should have.</li>
+            </ol>
+
+            {/* ------------------- Appeals ------------------- */}
+
+            <h3>Appeals</h3>
+
+            <p className="mb-5">
+              Only permanent actions (such as bans) may be appealed. To appeal a
+              decision, please contact an organizer listed above with your
+              appeal and freeCodeCamp Nashville will review the case. License
+            </p>
+
+            <p className="mb-5">
+              This code of conduct was originally derived from the{' '}
+              <a href="https://pynash.org/code-of-conduct">
+                PyNash Code of Conduct.
+              </a>
+            </p>
+
+            <p className="mb-5">
+              The freeCodeCamp Nashville Code of Conduct is similarly licensed
+              under a{' '}
+              <a href="https://creativecommons.org/licenses/by/3.0/">
+                Creative Commons Attribution 3.0 Unported License.
+              </a>
+            </p>
+          </div>
+        </div>
+      </section>
+    </Layout>
+  );
+}
+
+export const query = graphql`
+  query {
+    avatarSethAlexanderImage: file(relativePath: { eq: "Seth-Alexander.jpg" }) {
+      childImageSharp {
+        fluid(maxWidth: 260, quality: 100) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    avatarAlexThomasImage: file(relativePath: { eq: "Alex-Thomas.jpg" }) {
+      childImageSharp {
+        fluid(maxWidth: 260, quality: 100) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
```

Comment:
Old code?

Reply:
Removed the Old Grapql code. Nice caught. Thank you!